CHANGELOG.md

```diff
@@ -6,6 +6,17 @@
 
 ### CLI
 
+- Explicitly disable administrator rights for the native binary jumper on Windows. When a filename contains "install", "setup", "update", or "patch" Windows thinks it's an installer and requires administrator rights to start the binary
+
+## 2.3.1
+
+### CLI
+- Take into account peer dependency defaults when inheriting workspace peer dependencies in the node_modules linker
```

Comment:
@larixer More or less copied your commit message

Reply:
Its fine, thank you!

Reply:
It reads like someone is drilling through your head with hammerdrill, exactly the impression I was after

Reply:
For changelogs, it's fine if you stay vague and high-level: "Fixes an hoisting problem that could happen when workspaces listed a package as both a peer dependency and a regular one". People reading those usually don't care about the implementation, they just want to know if they are/will be impacted by the change.